toptal/picasso
# chore: sync packages

[FX-2190]

### Description

* Use up-to-date inner packages of monorepo.
* Setup syncpack to ignore peerDependencies

### How to test

1. manually change version of `@toptal/picasso-shared` in `picasso/package.json -> dependencies`
2. run `yarn syncpack:list` should not show any sync issues

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[FX-2190]: https://toptal-core.atlassian.net/browse/FX-2190?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
